### DISCUSSION TITLE
How to monitor an API endpoint?

### DISCUSSION THREAD

**Discussion by hdsfgsgrlsgtw12** [ccfos/nightingale#1418] [Q&A]
麻烦问下，例如http://xxx.xxx.xxx.xx/view/ask/这种接口的监控，是需要自己写脚本嘛

**Comment by UlricQin** [comment]
可以使用categraf的http_response插件，或者telegraf的http_response插件，或者blackbox-exporter

**Comment by zhangqingwen** [comment]
对 post 请求的支持怎么样，有些核心的 post 也想监控下
